.github/workflows/tests.yml

```diff
@@ -45,3 +47,42 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           file: ./coverage.xml
+
+  cpp-tests:
+    name: C++ tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.4.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout The Walrus
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get -y -q update
+          sudo apt-get -y -q install cmake gcc libomp-dev
+
+      - name: Install Google Test
+        working_directory: ~/googletest
```

Comment:
Seems to be complaining about this line:
![Screen Shot 2021-10-29 at 5 36 12 PM](https://user-images.githubusercontent.com/6934626/139504425-405b1f35-f006-4d56-a898-74130d069a2c.png)

Reply:
Thanks! I didn't catch that one 🔬